src/cf-forms/src/atoms/select.less

```diff
@@ -0,0 +1,68 @@
+.a-select {
+    position: relative;
+    border: 1px solid @select-border;
+
+    select {
+        height: unit( 30px / @base-font-size-px, em );
+        width: 100%;
+        padding: unit( 4px / @base-font-size-px, em )
+                 unit( 6px / @base-font-size-px, em );
+        border: 0;
+        outline: 2px solid transparent;
+        appearance: none;
+        background-color: @input-bg;
+        border-radius: 0;
+        color: @text;
+
+        &:hover,
+        &:active,
+        &:focus {
+            outline-color: @input-border__focused;
+            outline-offset: 0;
+        }
+    }
+
+    select[disabled] {
+        color: @input-text__disabled;
+        background-color: @input-bg__disabled;
+        cursor: not-allowed;
+
+        &:hover {
+            outline: none;
+        }
+    }
+
+    select option:disabled {
+        color: @input-disabled;
+    }
+
+    &:after {
+        width: unit( 30px / @base-font-size-px, em );
+        height: unit( 30px / @base-font-size-px, em );
```

Comment:
Line-height would still need to change, so I've gone ahead and removed this rule since line-height is really setting the height here. d0c16c4